core-bundle/src/Migration/Version500/RssToFeedReaderMigration.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration\Version500;
+
+use Contao\CoreBundle\Migration\AbstractMigration;
+use Contao\CoreBundle\Migration\MigrationResult;
+use Doctrine\DBAL\Connection;
+
+class RssToFeedReaderMigration extends AbstractMigration
+{
+    public function __construct(private Connection $connection)
+    {
+    }
+
+    public function shouldRun(): bool
+    {
+        $schemaManager = $this->connection->createSchemaManager();
+        $columns = $schemaManager->listTableColumns('tl_module');
```

Comment:
you should probably check if the table exists at all (e.g. on a fresh installation). Also make sure `feed_urls` and `feed_cache` do not exist.

Reply:
Fixed in [03843d0](https://github.com/contao/contao/pull/4847/commits/03843d08211a783cc7297133702e5e55ae0ffdf3)